contracts/ERC721.sol

```diff
@@ -0,0 +1,140 @@
+pragma solidity ^0.4.18;
+import './ERC721Interface.sol';
+
+
+/// @title Interface for contracts conforming to ERC-721: Non-Fungible Tokens
+/// @author Dieter Shirley <dete@axiomzen.co> (https://github.com/dete)
+// https://github.com/ethereum/EIPs/issues/721
+// Derived from https://github.com/decentraland/land/blob/master/contracts/BasicNFT.sol
+// & https://github.com/axiomzen/cryptokitties-bounty/blob/master/contracts/KittyOwnership.sol
+// & https://github.com/axiomzen/cryptokitties-bounty/blob/master/contracts/KittyBase.sol
+contract ERC721 is ERC721Interface {
+
+    uint256 public totalSupply;
+    string public name;
+    string public symbol;
+
+    // Array of owned tokens for a user
+    mapping(address => uint256[]) public ownedTokens;
+    mapping(address => uint256) public balances;
+    mapping(uint256 => uint256) public tokenIndexInOwnerArray;
+
+    // Mapping from token ID to owner
+    mapping(uint256 => address) public tokenOwner;
+
+    // Allowed transfers for a token (only one at a time)
+    mapping(uint256 => address) public allowed;
+
+    // Metadata associated with each token
+    mapping(uint256 => string) public tokenMetadata;
+
+
+    modifier tokenExists(uint256 _tokenId) {
+        require(tokenOwner[_tokenId] != 0);
+        _;
+    }
+
+    //how many badges of a specific release someone owns
+    function balanceOf(address _owner) public view returns (uint256 balance) {
```

Comment:
I propose taking the named var return out.

Changing this: `returns (uint256 balance)`
To this: `returns (uint256)`

Since the `balance` var is not even used